src/style.css

```diff
@@ -0,0 +1,142 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+}
+
+body {
+  background-color: #f6f6f6;
+  font-family: 'Open Sans', 'Lucida Grande', tahoma, verdana, arial, sans-serif;
+  text-align: center;
+  font-weight: 300;
+}
+
+main {
+  padding: 0 10%;
+}
+
+#todo-list-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin: 25% 0;
+
+  /* padding: 0 20%; */
+  overflow: hidden;
+  cursor: pointer;
+
+  /* width: 60%; */
+  transition: padding 300ms ease-in-out, max-height 500ms ease-in-out;
```

Comment:
- [ ] Same here, Please do not leave unused code as a comment in your codebase. this will make your code much noisy, It is always a good practice to keep your code clean from such comments. 😉 👍🏻